Zhuang/README.md

```diff
@@ -0,0 +1,33 @@
+# Yan Zhuang student-developer-setup
+
+1) Created fork on GitHub
+
+2) Created README.md
+
+3) Pushed to GitHub, created pull request
+
+4) Using Pycharm (Working on Windows System)
+
+5) Created `hello.py` as a "hello world" program; can be built via the `build.sh` script.
+6-7) Created requirement.txt and add pre-commit as required package
```

Comment:
There's no `requirement.txt` on the branch anymore, so these instructions aren't accurate. These need to get updated.